plux/build/setuptools.py

```diff
@@ -39,20 +41,31 @@ def _ensure_dist_info(self, *args, **kwargs):
 class plugins(InfoCommon, setuptools.Command):
     description = "Discover plux plugins and store them in .egg_info"
 
-    user_options = [
-        # TODO
+    user_options: t.ClassVar[t.List[t.Tuple[str, str, str]]] = [
+        ('exclude=', 'e', "exclude those files when discovering plugins"),
```

Comment:
perhaps again we can be a bit more verbose here with the help

```suggestion
        ('exclude=', 'e', "a sequence of paths to exclude; '*' can be used as a wildcard in the names. 'foo.*' will exclude all subpackages of 'foo' (but not 'foo' itself)."),
```